app/src/androidTest/java/sdp/moneyrun/database/RiddlesDatabaseTest.java

```diff
@@ -31,11 +31,12 @@ public static void setPersistence() {
     @Test
     public void testGetInstance() {
         RiddlesDatabase db = RiddlesDatabase.createInstance(context);
-        assertEquals(RiddlesDatabase.getInstance(), db);
+        Assert.assertEquals(RiddlesDatabase.getInstance(), db);
         RiddlesDatabase.reset();
 
     }
 
+    @Test
     public void testGetInstanceOnNonExistentThrowsException() {
```

Comment:
I agree, I am not sure who used that notation but it is not very good practice. Unfortunately, it is not very realistic for someone to change all of this as it would take a huge amount of time, I think the best solution is for each to fix their tests.